src/borg/archiver.py

```diff
@@ -2665,42 +2687,40 @@ def do_break_lock(self, args, repository):
             # note: using = is essential to avoid cmdline argument parsing issues.
             borg create --pattern=+pics/2018/good --pattern=-pics/2018 repo::arch pics
 
+            # backup only JPG/JPEG files (case insensitive) in all home directories:
+            borg create --pattern 're:\\.jpe?g(?i)' repo::arch /home
+
             # use a file with patterns:
-            borg create --patterns-from patterns.lst repo::arch
+            borg create --patterns-from patterns.lst repo::arch /
 
         The patterns.lst file could look like that::
 
-            # "sh:" pattern style is the default, so the following line is not needed:
+            # sh: pattern style is the default, so the following line is not needed:
```

Comment:
is that an improvement? i guess the quotes were there intentionally because the colon is not punctuation in that sentence.

Reply:
ok, reverting